Cassiopee/Envs/sh_Cassiopee_local

```diff
@@ -526,6 +526,33 @@ elif [ "$MAC" = "juno" ]; then
     unset $(env | grep SLURM | cut -d'=' -f 1)
     export OMP_PLACES=cores
 
+elif [ "$MAC" = "junoMain" ]; then
```

Comment:
Please delete, prods that are specific to a single user should be added here imo.

Reply:
Apologies, I had not seen/read previous comments...